packages/ag-grid-theme/css/_salt-ag-theme-icon-font.scss

```diff
@@ -58,14 +100,14 @@ $icons: (
 }
 
 [class^="ag-theme-salt"] .ag-checkbox-input-wrapper::after {
-  font-family: "jpmuitk-icons";
+  font-family: "salt-icons";
   content: "";
 }
 
 [class^="ag-theme-salt"] .ag-checkbox-input-wrapper.ag-checked::after {
-  content: "\e95b";
+  content: "\ea2e";
 }
 
 [class^="ag-theme-salt"] .ag-checkbox-input-wrapper.ag-indeterminate::after {
-  content: "\e94e";
+  content: "\ea10";
```

Comment:
This appears to be full width stroke, can you verify this with design? 
[CheckboxSelection story](https://91393344.saltdesignsystem-storybook.pages.dev/?path=/docs/data-grid-ag-grid-theme--checkbox-selection)

Reply:
this is the same as uitk